CSharpSourceCode/SubModule.cs

```diff
@@ -128,9 +129,12 @@ protected override void OnGameStart(Game game, IGameStarter gameStarterObject)
             else if(game.GameType is Campaign)
             {
                 CampaignGameStarter starter = gameStarterObject as CampaignGameStarter;
+                PartyAttributeManager partyAttributeManager = new PartyAttributeManager();
+                starter.CampaignBehaviors.Add(partyAttributeManager);
                 starter.CampaignBehaviors.RemoveAllOfType(typeof(BackstoryCampaignBehavior));
                 starter.Models.RemoveAllOfType(typeof(CompanionHiringPriceCalculationModel));
                 starter.AddModel(new TowCompanionHiringPriceCalculationModel());
+                starter.AddModel(new CustomBattleMoralModel.TOWCampaignBattleMoralModel());
```

Comment:
Small thing. Moral vs Morale. Spelling.

Morale is fighting spirit or willingness to carry through with some action.
A moral is relating to the idea of right or wrong.

Reply:
done